python/src/CrossEntropyResult_doc.i.in

```diff
@@ -0,0 +1,70 @@
+%feature("docstring") OT::CrossEntropyResult
+"Cross Entropy result.
+
+See also
+--------
+CrossEntropyImportanceSampling, PhysicalSpaceCrossEntropyImportanceSampling, StandardSpaceCrossEntropyImportanceSampling, SimulationResult"
+
+
+// ---------------------------------------------------------------------------
+
+%feature("docstring") OT::CrossEntropyResult::getAuxiliaryDistribution
+"Accessor of the auxiliary distribution at the final Cross Entropy algorithm step.
```

Comment:
I think it should be "accessor to", here and in several other docstrings in this file.

Reply:
Yes, of course.